filters/auth/oauth.go

```diff
@@ -234,13 +395,13 @@ func (s *tokeninfoSpec) Name() string {
 }
 
 // CreateFilter creates an auth filter. All arguments have to be
-// strings. Depending on the variant of the auth filter, the arguments
+// strings. Depending on the variant of the auth tokeninfoFilter, the arguments
 // represent scopes or key-value pairs to be checked in the tokeninfo
 // response. How scopes or key value pairs are checked is based on the
 // type. The shown example for checkOAuthTokeninfoAllScopes will grant
 // access only to tokens, that have scopes read-x and write-y:
 //
-//     s.CreateFilter(read-x", "write-y")
+//     s.CreateTokeninfoFilter(read-x", "write-y")
```

Comment:
this doc entry should stay s.CreateFilter

Reply:
oh this was a bigger replace Filter with TokeninfoFilter ...